src/components/api-request.js

```diff
@@ -704,7 +704,7 @@ export default class ApiRequest extends LitElement {
     const responseFormat = this.responseHeaders.includes('json') ? 'json' : (this.responseHeaders.includes('html') || this.responseHeaders.includes('xml')) ? 'html' : '';
     return html`
       <div class="row" style="font-size:var(--font-size-small); margin:5px 0">
-        <div class="response-message ${this.responseStatus}">Response Status: ${this.responseMessage}</div>
+        <div class="response-message ${this.responseStatus}">Response Status: ${this.responseMessage} in ${this.elapsedMs}ms</div>
```

Comment:
Just to make this a bit more clear I'm thinking about this small change
```suggestion
        <div class="response-message ${this.responseStatus}">Response Status: ${this.responseMessage}
           ${this.elapsedMs ? html`<span><br>Execution Time: ${this.elapsedMs}ms</span>` : ''}
         </div>
```

Reply:
Done